.github/workflows/cmake.yml

```diff
@@ -524,7 +534,7 @@ jobs:
           -DLLVM_ENABLE_PROJECTS="libcxx;libcxxabi" \
           -DLLVM_USE_SANITIZER=MemoryWithOrigins \
           -DLLVM_LIBC_ENABLE_LINTING=OFF
-        cmake --build llvm-project/build -- cxx cxxabi
+        cmake --build llvm-project/build -j2 -- cxx cxxabi
```

Comment:
Do we want `-j ${{ matrix.parallels-jobs || '2' }}` here?

Reply:
I did consider that, but to be really useful it would have to be split into two settings, one for compile and one for tests. (There seems to be more to gain by overloading during testing, likely because of more io-wait and context switching and less cpu-load)

But I figured we might not want to do more than 2 compiles at a time in any case, since it has a tendency to make the build log harder to read when there is an error somewhere. Also, there is not much to be gained from going above 2.
The GHA 2-core VMs would possibly get a very slight boost at 3, if there is any io-wait time in the other threads, but likely those two cores are already contending with other VMs for cpu-time.
The s390x 4-core VMs would probably get much faster at 4-5 threads, but only during the already quite fast compile, and the s390x runs already complete long before any of the other workflows.

So I figured it was not really worth the effort to make it configurable just to save a little bit of time where it doesn't really help us the first place. I am not opposed to a PR making it configurable though.